src/modules/draft/draftApiInterfaces.ts

```diff
@@ -0,0 +1,316 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+interface Author {
+  type: string;
+  name: string;
+}
```

Comment:
det er to av denne, skal de vokse til å bli ulike? Eller trenger du egentlig bare en? :) 
Den andre er i articleApiInterfaces

Reply:
Stemmer, både article og draft bruker denne og den er likt for begge de to. Skal slå de sammen :+1: